concert/app/test/on_event_stage_changed/test_on_event_stage_changed.py

```diff
@@ -0,0 +1,104 @@
+import json
+import os
+import pytest
+from botocore.exceptions import ClientError
+from botocore.stub import Stubber
+from src.functions.on_event_stage_changed.index import handler, sns, dynamodb
+
+CALLBACK_SNS_ARN = os.environ["CALLBACK_GLOBAL_SNS_TOPIC_ARN"]
+TABLE_NAME = os.environ.get("DYNAMODB_TABLE_NAME")
+
+SUCCESS_RESPONSE = {"ResponseMetadata": {"HTTPStatusCode": 200}}
+
+
+def get_event(stageId="STAGE_CHOICE_COLOR_EMOTION", otherAttributes=None):
+    data = {"stageId": stageId}
+    if otherAttributes:
+        for key, value in otherAttributes.items():
+            data[key] = value
+    return {"body": json.dumps({"event": "EVENT_STAGE_CHANGED", "data": (data)})}
+
+
+def test_event_stage_changed():
+    stageId = "STAGE_CHOICE_COLOR_EMOTION"
+    otherAttributes = {
+        "startTime": "2019-05-14T21:20:03.000Z",
+        "endTime": "2019-05-14T21:21:33.000Z",
+        "frequency": 20,
+    }
+    event = get_event(stageId, otherAttributes)
+
+    expectedMessage = json.dumps(
+        {
+            "event": "EVENT_STAGE_CHANGED",
+            "data": {
+                "stageId": stageId,
+                "startTime": otherAttributes["startTime"],
+                "endTime": otherAttributes["endTime"],
+                "frequency": otherAttributes["frequency"],
+            },
+        }
+    )
+
+    sns_expected_params = {
+        "TopicArn": CALLBACK_SNS_ARN,
+        "Message": expectedMessage,
+        "MessageStructure": "string",
+        "MessageAttributes": {
```

Comment:
We actually shouldn't be using MessageAttributes anymore. That was based on using topic filters, which we discovered we could not do because of account limits.

Reply:
Removed.